javascript-constructors/new-objects.js

```diff
@@ -0,0 +1,6 @@
+var anObject = new Object();
+var anArray = new Array;
+var aFunction = new Function;
+var aString = new String;
+var aNumber = new Number;
+var aBoolean = new Boolean;
```

Comment:
@EvanS-Hong Please examine their values, types, and include a screenshot with your Pull Request:

---

![image](https://user-images.githubusercontent.com/16143771/220429046-16102dc9-44dc-4e04-8c90-d687be979bb0.png)